curriculum/challenges/english/25-front-end-development/quiz-css-flexbox/66ed8fe7f45ce3ece4053eb2.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
-Placeholder question
+Which value of `justify-content` packs flex items toward the start of the container?
 
 #### --distractors--
 
-Placeholder distractor 1
+`center`
 
 ---
 
-Placeholder distractor 2
+`space-between`
 
 ---
 
-Placeholder distractor 3
+`space-around`
 
 #### --answer--
 
-Placeholder answer
+`flex-start`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does `space-between` do in `justify-content`?
 
 #### --distractors--
 
-Placeholder distractor 1
+Distributes space between items, with equal space on both sides of each item
 
 ---
 
-Placeholder distractor 2
+Centers items along the main axis
 
 ---
 
-Placeholder distractor 3
+Aligns items at the end of the container
 
 #### --answer--
 
-Placeholder answer
+Distributes space between items, with no space at the start or end
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which value of `flex-wrap` wraps items in reverse order on multiple lines?
 
 #### --distractors--
 
-Placeholder distractor 1
+`nowrap`
 
 ---
 
-Placeholder distractor 2
+`wrap`
 
 ---
 
-Placeholder distractor 3
+`reverse-wrap`
 
 #### --answer--
 
-Placeholder answer
+`wrap-reverse`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which property is responsible for controlling the layout direction of flex items?
 
 #### --distractors--
 
-Placeholder distractor 1
+`justify-content`
 
 ---
 
-Placeholder distractor 2
+`flex-wrap`
 
 ---
 
-Placeholder distractor 3
+`align-items`
 
 #### --answer--
 
-Placeholder answer
+`flex-direction`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which value of `align-items` aligns flex items at the end of the cross axis?
 
 #### --distractors--
 
-Placeholder distractor 1
+`flex-start`
 
 ---
 
-Placeholder distractor 2
+`baseline`
 
 ---
 
-Placeholder distractor 3
+`center`
 
 #### --answer--
 
-Placeholder answer
+`flex-end`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the default value of `align-items`?
 
 #### --distractors--
 
-Placeholder distractor 1
+`flex-start`
 
 ---
 
-Placeholder distractor 2
+`center`
 
 ---
 
-Placeholder distractor 3
+`flex-end`
 
 #### --answer--
 
-Placeholder answer
+`stretch`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+If you want to ensure there is no wrapping of flex items, which `flex-wrap` value should you use?
 
 #### --distractors--
 
-Placeholder distractor 1
+`wrap-reverse`
 
 ---
 
-Placeholder distractor 2
+`wrap`
 
 ---
 
-Placeholder distractor 3
+`flex-end`
 
 #### --answer--
 
-Placeholder answer
+`nowrap`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which value of `justify-content` evenly distributes items with space around them, including at the start and end?
 
 #### --distractors--
 
-Placeholder distractor 1
+`space-between`
 
 ---
 
-Placeholder distractor 2
+`space-around`
 
 ---
 
-Placeholder distractor 3
+`flex-start`
 
 #### --answer--
 
-Placeholder answer
+`space-evenly`
```

Comment:
it is also an answer on line 80, this question is a duplicate and needs to be changed to something else